### ISSUE TITLE
attach hooks to raw collections

### ISSUE THREAD

**Issue by micktaiwan** [Meteor-Community-Packages/meteor-collection-hooks#276]
What would be a simple way to attach hooks to raw collections (except by redefining collection.find, etc...)
I really would like to use this package on mongodb raw collections.

**Comment by mathiasbie** [comment]
Hello! 
Sorry for digging up such an old topic.
I was wondering if you found a solution to the above issue? 
Sometimes, I want to perform bulkInsert/write/... operations on collection A and would like to retrieve everything I've inserted/modified to update counters in collection B (which depends on A)."

What are your thoughts on this?

Best regards,
Mathias

**Comment by micktaiwan** [comment]
No, I just overrided find, findOne, etc by hand. So I did what I wanted to avoid (redefining collection.find)

**Comment by mathiasbie** [comment]
OK thanks for the quick answer !
